OracleJava/java-11/build.sh

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
You need to add Copy right and UPL
#Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
#
#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.

Reply:
Done